### ISSUE TITLE
cyclomatic_complexity wrong estimate ?

### ISSUE THREAD

**Issue by petarbelokonski** [realm/SwiftLint#3252]
### New Issue Checklist

- [x] using version 0.38.0
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

I might be missing something but this code:

```
    func testCYC(bool: Bool) {
        if bool {
            print("it is true")
        }
        print("it is default")
    }
```

has 2 paths of execution 

![Screenshot 2020-06-26 at 19 00 10](https://user-images.githubusercontent.com/5446543/85877077-46b3cd00-b7df-11ea-82dc-ba2e688436b4.png)

and if we follow the [cyclomatic complexity formula](https://en.wikipedia.org/wiki/Cyclomatic_complexity) where M = E - N + 2*P, this produces M = 2. However when I change the `cyclomatic_complexity` warning to 1, no warning is displayed. I feel like SwiftLint treats this to have a complexity of 1 and I am not sure why. 

Is this a bug ?

For the record: https://github.com/terryyin/lizard produces M = 2 for the same code snippet which is what I'd expect.

```yml
cyclomatic_complexity:
  warning: 1
```

**Comment by petarbelokonski** [comment]
@stale: reopen

**Comment by jpsim** [comment]
I'm not completely sure, but I'm inclined to agree with you @petarbelokonski, but I'm not seeing the problem in the way we measure cyclomatic complexity. Maybe you can see it: https://github.com/realm/SwiftLint/blob/0.42.0/Source/SwiftLintFramework/Rules/Metrics/CyclomaticComplexityRule.swift
